packages/abi-coder/src/encoding/coders/EnumCoder.ts

```diff
@@ -21,8 +21,8 @@ export class EnumCoder<TCoders extends Record<string, Coder>> extends Coder<
   InputValueOf<TCoders>,
   DecodedValueOf<TCoders>
 > {
-  name: string;
-  coders: TCoders;
+  public override name: string;
+  public coders: TCoders;
```

Comment:
```suggestion
  override name: string;
  coders: TCoders;
```

Why change the access visibility here?